x/emissions/errors.go

```diff
@@ -55,16 +55,6 @@ var (
 	ErrNotInTopicCreationWhitelist                       = errors.Register(ModuleName, 50, "not in topic creation whitelist")
 	ErrNotInWeightSettingWhitelist                       = errors.Register(ModuleName, 51, "not in topic weight setting whitelist")
 	ErrTopicNotEnoughDemand                              = errors.Register(ModuleName, 52, "topic not enough demand")
-	ErrSetParamsVersion                                  = errors.Register(ModuleName, 53, "Error Setting Params: Version")
-	ErrSetParamsEpochLength                              = errors.Register(ModuleName, 54, "Error Setting Params: EpochLength")
-	ErrSetParamsEmissionsPerEpoch                        = errors.Register(ModuleName, 55, "Error Setting Params: EmissionsPerEpoch")
-	ErrSetParamsMinTopicUnmetDemand                      = errors.Register(ModuleName, 56, "Error Setting Params: MinTopicUnmetDemand")
-	ErrSetParamsMaxTopicsPerBlock                        = errors.Register(ModuleName, 57, "Error Setting Params: MaxTopicsPerBlock")
```

Comment:
why are these being removed? no longer used?

Reply:
spoke about this over slack